src/Symfony/Component/HttpFoundation/File/UploadedFile.php

```diff
@@ -145,13 +145,13 @@ public function guessClientExtension()
      * It is extracted from the request from which the file has been uploaded.
      * Then it should not be considered as a safe value.
      *
-     * @deprecated since 4.1 will be removed in 5.0 use getSize() instead.
+     * @deprecated since Symfony 4.1, to be removed in 5.0. Use getSize() instead.
```

Comment:
Same here, should be moved to another PR.

Reply:
Luckily I saw you already did that in #26225